tests/Refactor/Xml/XmlWriterTest.php

```diff
@@ -0,0 +1,84 @@
+<?php
+namespace DemandwareXml\Test\Refactor\Xml;
+
+use DemandwareXml\Refactor\Xml\XmlWriter;
+use PHPUnit\Framework\TestCase;
+
+class XmlWriterTest extends TestCase
+{
+    private function getMemoryXmlWriter(): XmlWriter
+    {
+        $xml = new XmlWriter;
+        $xml->openMemory();
+        $xml->setIndentDefaults();
+
+        return $xml;
+    }
+
+    private function getFileXmlWriter(string $filename): XmlWriter
+    {
+        $xml = new XmlWriter;
+        $xml->openFile($filename);
+        $xml->setIndentDefaults();
+
+        return $xml;
+    }
+
+    public function test_simple_catalog_memory(): void
+    {
+        $xml = $this->getMemoryXmlWriter();
+        $xml->startDocument('1.0', 'UTF-8');
```

Comment:
Could we leave out these arguments or are we testing that they're omitted from the output?

Reply:
@WillJW The document tag is in the output, the assertion just understands how to compare without it I would assume. I'd leave things as they are, as it's clear we're making a valid XML document this way.